CocoIDE/cocol.py

```diff
@@ -12,32 +12,33 @@
 
 try:
     input = raw_input  # Python 3 style input()
-except:
+except NameError:
     pass  # Running on Python 3
```

Comment:
This whole try/except should just be removed

Reply:
I thought python 2 was still to be supported.

Reply:
In nothing else your own commits broke Python 2 support, but no given 2 has been EOL for some time we've moved on